CONTRIBUTING.md

```diff
@@ -17,6 +17,13 @@ that link in issues.
 - `cd bootstrap-vue`
 - Run `yarn install` to get all dependencies installed
 
+## Online one-click setup for Contributing
+
+Contribute to BootstrapVue, using a fully featured online development environment; cloned repo,
+pre-installed dependencies, running web server and rollup watching for changes.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
+
```

Comment:
I am thinking we need to have a bit more explanation as to how the editor works... forking, and pushing changes to a PR.

Reply:
Ok! Thanks Should I mention those below the gitpod button in `CONTRIBUTING.md`?

Reply:
As it is not overly clear when first clicking the button

Reply:
Probably some basic how-to bullets.  and a link to the documentation for more detail on using it

Reply:
Ok. Thanks, I will do this later today in the evening.

Reply:
And we may add the information to our pull request template...

Reply:
Also noticed a few errors when rendering the preview:

```
https://3000-b99d7bd4-3572-46d1-951c-1530acfeb0b1.ws-us02.gitpod.io/_loading/sse net::ERR_HTTP2_PROTOCOL_ERROR 200
```

Reply:
Also noticed sluggishness on clicking router links in the review... takes about 30 seconds to see the route change

Reply:
it might be that nuxt needs more than 1G of ram to render.

Reply:
> And that the preview take a bit of time to render. Not sure if it is a memory limit issue or not.

Thanks! A Gitpod workspace can use up to 12GB of RAM, which should be enough for nuxt.js but it would interesting to figure out why the route changing is slow and to make if faster if possible.